### PR TITLE
Issue #2949 Fixed typo in cmd/minishift/cmd/util/oc.go

### LINKED ISSUE #2949: Typo in cmd/minishift/cmd/util/oc.go
There's a typo in **_cmd/minishift/cmd/util/oc.go_**:

```go
fmt.Printf("-- Pulling the Openshift Container Image ")
```

"Openshift" should be "OpenShift" here.

Likewise here:

```go
return fmt.Errorf("Error pulling the openshift container image: %v", err)
```

### PR DESCRIPTION
**Pull request by Sajfer**
Fixes #2949 

Corrects typo in cmd/minishift/cmd/util/oc.go

### CODE REVIEW

**Comment by anjannath** [comment]
ok to test

File: cmd/minishift/cmd/util/oc.go [hunk 1]
```diff
@@ -152,12 +152,12 @@ func PullOpenshiftImageAndCopyOcBinary(dockerCommander docker.DockerCommander, r
...
 	imageExist, err := dockerCommander.IsImageExist(minishiftConstants.GetOpenshiftImageToFetchOC(requestedOpenShiftVersion))
 	if !imageExist {
-		fmt.Printf("-- Pulling the Openshift Container Image ")
+		fmt.Printf("-- Pulling the OpenShift Container Image ")
 		progressDots := progressdots.New()
 		progressDots.Start()
 		_, err = dockerCommander.Pull(minishiftConstants.GetOpenshiftImageToFetchOC(requestedOpenShiftVersion))
 		if err != nil {
-			return fmt.Errorf("Error pulling the openshift container image: %v", err)
+			return fmt.Errorf("Error pulling the OpenShift container image: %v", err)
```
Comment (praveenkumar) [inline cmd/minishift/cmd/util/oc.go]: 👍

File: cmd/minishift/cmd/util/oc.go [hunk 2]
```diff
@@ -152,12 +152,12 @@ func PullOpenshiftImageAndCopyOcBinary(dockerCommander docker.DockerCommander, r
 	// We need to make sure if images are already exist from the cache then don't pull it again.
 	imageExist, err := dockerCommander.IsImageExist(minishiftConstants.GetOpenshiftImageToFetchOC(requestedOpenShiftVersion))
 	if !imageExist {
-		fmt.Printf("-- Pulling the Openshift Container Image ")
+		fmt.Printf("-- Pulling the OpenShift Container Image ")
```
Comment (praveenkumar) [inline cmd/minishift/cmd/util/oc.go]: 👍

**Comment by praveenkumar** [comment]
@Sajfer Wait till CI pass, hope it doesn't have those messages hard coded as part of error then you need to changes those messages also.

**Comment by praveenkumar** [comment]
CI failure was as expected.

**Pull request merged**